core/src/test/scala/io/finch/EndToEndSpec.scala

```diff
@@ -53,4 +55,15 @@ class EndToEndSpec extends FinchSpec {
     rep.contentString shouldBe "bar"
     rep.status shouldBe Status.Created
   }
+
+  it should "convert Endpoint[AsyncStream[Buf]] to service" in {
```

Comment:
It seems to be impossible to get ```AsyncStream[Buf]``` from ```ToResponse#apply``` since it returns ```Response``` and handles stream writing internally.
The only way I can think of is to copy output to ```Request#contentString``` upon writing, but I really doubt that it's what we want..